homeassistant/components/doorbird/__init__.py

```diff
@@ -137,16 +95,126 @@ def _reset_device_favorites_handler(event):
             for favorite_id in favorites[favorite_type]:
                 doorstation.device.delete_favorite(favorite_type, favorite_id)
 
-    hass.bus.listen(RESET_DEVICE_FAVORITES, _reset_device_favorites_handler)
+    hass.bus.async_listen(RESET_DEVICE_FAVORITES, _reset_device_favorites_handler)
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up DoorBird from a config entry."""
+
+    _async_import_options_from_data_if_missing(hass, entry)
+
+    doorstation_config = entry.data
+    doorstation_options = entry.options
+    config_entry_id = entry.entry_id
+
+    device_ip = doorstation_config[CONF_HOST]
+    username = doorstation_config[CONF_USERNAME]
+    password = doorstation_config[CONF_PASSWORD]
+
+    device = DoorBird(device_ip, username, password)
+    try:
+        status = await hass.async_add_executor_job(device.ready)
+        info = await hass.async_add_executor_job(device.info)
+    except urllib.error.HTTPError:
+        if urllib.error.HTTPError.code == 401:
```

Comment:
This doens't seem right. `urllib.error.HTTPError` is the class. You should get the error instance inside the `except`.
```suggestion
    except urllib.error.HTTPError as err:
        if err.code == 401:
```

Reply:
It's not.  I fixed the other one but missed this.

Reply:
Thanks for noticing that. I was a bit distracted today with COVID-19 crazy.